pattern-combine/apps/split-combine/src/main/java/com/redhat/training/combine/CombineRouteBuilder.java

```diff
@@ -0,0 +1,20 @@
+package com.redhat.training.combine;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.processor.aggregate.AggregationStrategy;
+import org.apache.camel.builder.RouteBuilder;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CombineRouteBuilder extends RouteBuilder {
+
+    private static final int BATCH_COMPLETION_INTERVAL = 10;
+    private static String separator = System.getProperty("line.separator");
+
+    @Override
+    public void configure() throws Exception {
+        from( "file:orders/incoming?noop=true" )
+                // Split and Aggregate
```

Comment:
Suggested change: normally we use this `// TODO: ...` format. 
Also, I was a bit unsure about were to add the `completionSize` call, so it might be interesting to add another comment indicating where to add this call

```suggestion
                // TODO: Split and Aggregate
                // TODO: call completionSize
```